ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/logger/Slf4jAuditLogger.java

```diff
@@ -100,6 +100,7 @@ public Slf4jAuditLogger(Map<String, String> parameters, String loggerName)
                .put("STATUS", AuditEntry::getStatus)
                .put("OPERATION", entry -> entry.getOperation().getOperationString())
                .put("OPERATION_NAKED", entry -> entry.getOperation().getNakedOperationString())
+               .put("SINGLE_LINE_OPERATION", Slf4jAuditLogger::singleLineStatement)
```

Comment:
Would it make sense to add corresponding functionality in the ecLog tool as well?

Reply:
After starting to implement it I'm not so sure. I can see the use case for OPERATION_NAKED as that could reduce the amount of data stored.

This is for visual purposes (e.g. when parsing logs) and as such I wonder if it wouldn't be better to have it as an option for printing in the tool instead?

Reply:
Right, actually, that was what I meant to say.